paths/custom_metadata_properties/index.yaml

```diff
@@ -10,7 +10,7 @@ tags:
 parameters:
 - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
 - "$ref": "../../parameters.yaml#/account_id"
-- "$ref": "../../parameters.yaml#/data_type"
+- "$ref": "../../parameters.yaml#/custom_metadata_data_type"
```

Comment:
I'm curious, why is there a data_type parameter? is it for filtering properties only for the given type? what was the use case for adding this?